src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const promise1 = new Promise((resolve, reject) => {
+  const timeout = setTimeout(reject, 3000);
+
+  document.addEventListener('click', () => {
+    clearTimeout(timeout);
+    resolve();
+  });
+});
+const promise2 = new Promise((resolve) => {
+  document.addEventListener('contextmenu', (even) => {
```

Comment:
Typo: The parameter `even` should be `event` to correctly reference the event object.